src/helpers/image-capture/multipart-file.ts

```diff
@@ -0,0 +1,17 @@
+import { Asset } from 'react-native-image-picker';
+
+interface IMultipartFile {
+  uri: string;
+  name: string;
+  type: string;
+}
+
+function assetToMultipartFile(asset: Asset): IMultipartFile {
+  return {
+    uri: asset.uri ?? asset.base64 ?? '',
```

Comment:
shouldn't base64 value be first in here?

Reply:
No